protocol/native/native.go

```diff
@@ -36,7 +38,28 @@ func ExecParser(toNoder ToNoder, path string, args ...string) (*Parser, error) {
 }
 
 func (p *Parser) ParseUAST(req *protocol.ParseUASTRequest) *protocol.ParseUASTResponse {
-	nativeResp, err := p.Client.ParseNativeAST(&ParseASTRequest{Content: req.Content})
+	// If the request encoding is Base64 try to decode it as UTF8. Leave it
+	// as Base64 if it isn't possible.
```

Comment:
I don't understand why we try to encode as UTF8 if we have explicitely been told that the encoding is base64, what's the reason?
As I understand it, it should be left to the native parser to handle this, I don't think this should be solved in a generic way.